### PR TITLE
Add a time-based registration limit

### PR DESCRIPTION
**Pull request by davidmz**
⚠ Introduces a new field in the config!

### CODE REVIEW

**Review by dsumin — APPROVED** [review]
LGTM

File: app/controllers/api/v1/UsersController.js [hunk 1]
```diff
@@ -23,6 +30,12 @@ export default class UsersController {
   static create = compose([
     inputSchemaRequired(userCreateInputSchema),
     async (ctx) => {
+      const registrationOpen = await ServerInfo.isRegistrationOpen();
+
+      if (!registrationOpen) {
+        throw new TooManyRequestsException('New user registrations are temporarily stopped');
```
Comment (dsumin) [inline app/controllers/api/v1/UsersController.js]: @davidmz 'New user registrations are temporarily suspended.'

**Review by dsumin — APPROVED** [review]
LGTM

File: app/controllers/api/v1/UsersController.js (hunk 1 above)
Comment (davidmz) [reply app/controllers/api/v1/UsersController.js]: ок

**Pull request merged**